shared/common-adapters/dropdown.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import * as React from 'react'
```

Comment:
we used to have a dropdown that was just used by login and wasn't 'generic' at all. now this is super generic